src/components/books/Book.jsx

```diff
@@ -17,39 +21,52 @@ const Book = ({ book }) => {
           <p className={styles.book__author}>{author}</p>
         </div>
         <div className={styles['book__btn-group']}>
-          <button type="button" className={styles.book__btn}>
+          <button
+            type="button"
+            className={styles.book__btn}
+          >
             Comments
           </button>
-          <button type="button" className={styles.book__btn}>
+          <button
+            type="button"
+            className={styles.book__btn}
+            onClick={handleRemoveBtnClick}
+          >
             Remove
           </button>
-          <button type="button" className={styles.book__btn}>
+          <button
+            type="button"
+            className={styles.book__btn}
+          >
             Edit
           </button>
         </div>
       </div>
       <div className={styles.book__right}>
         <div className={styles['book__progress-container']}>
           <div className={styles['book__progress-container']}>
-            <h4 className={styles['book__progress-title']}>64%</h4>
+            <h4 className={styles['book__progress-title']}>
+              {`${completed}%`}
+            </h4>
             <p className={styles['book__progress-complete']}>Completed</p>
           </div>
         </div>
-        <div className={styles['book__chapter-details']}>
+        {/* <div className={styles['book__chapter-details']}>
           <div className={styles['book__chapter-container']}>
             <span className={styles['book__chapter-name']}>
               Current Chapter
             </span>
-            <h4 className={styles['book__chapter-title']}>
-              {currentChapter}
-            </h4>
+            <h4 className={styles['book__chapter-title']}>{completed}</h4>
           </div>
           <div className={styles['book__chapter-btn-wrapper']}>
-            <button type="button" className={styles['book__chapter-btn']}>
+            <button
+              type="button"
+              className={styles['book__chapter-btn']}
+            >
               Update Progress
             </button>
           </div>
-        </div>
+        </div> */}
```

Comment:
- [ ] Please consider removing this commented code. Unused pieces of code make your codebase less readable and dirty. 🤓 🔧

Reply:
![image](https://user-images.githubusercontent.com/55840999/214172786-0a88e995-2c2f-4c57-b472-db9eca88a289.png)
As per requirement, categories and comments are NOT part of this step, so I commented them. @anagudelogu